replibyte/src/bin/example.rs

```diff
@@ -1,27 +1,14 @@
-use crate::bridge::s3::S3;
-use crate::config::{Config, ConnectionUri};
+use replibyte::bridge::s3::S3;
```

Comment:
I think maybe `cli.rs` is a better name, at some point we will want to add a cli.